Makefile

```diff
@@ -22,6 +33,17 @@ build-linux: go-mod
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o firmware-syncer -mod vendor
 	sha256sum firmware-syncer > firmware-syncer_checksum.txt
 
+build-image: build-linux
+	docker build --rm=true -f Dockerfile -t ${DOCKER_IMAGE}:latest  . \
+							 --label org.label-schema.schema-version=1.0 \
+							 --label org.label-schema.vcs-ref=$(GIT_COMMIT_FULL) \
+							 --label org.label-schema.vcs-url=$(REPO)
+
+## Build devel docker image
+build-image-devel: build-image
```

Comment:
can you rename this to `push-image-devel` similar to here https://github.com/metal-toolbox/flasher/blob/7b55d2571eabec60a9b6a7b174fa55cb74edb9ae/Makefile#L60

Reply:
Fixed in [95f5b33](https://github.com/metal-toolbox/firmware-syncer/pull/135/commits/95f5b33069b0bddbe4837323bc4c539765f8c6c1)